README.md

```diff
@@ -1,21 +1,36 @@
-# Dracula for [X](http://link-to-x.com)
+# Dracula for [Gedit](https://wiki.gnome.org/Apps/Gedit)
 
-> A dark theme for [X](http://link-to-x.com).
+> A dark theme for [Gedit](https://wiki.gnome.org/Apps/Gedit).
 
-![Screenshot](https://draculatheme.com/assets/img/screenshots/sublime.png)
+![Screenshot](http://i.imgur.com/paelbpq.png)
```

Comment:
Please keep the URL as it was but replace `sublime` with `gedit`. When the PR is merged in that we talked about for the screenshot the url will become valid.